tests/unit/test_crud.py

```diff
@@ -1076,3 +1084,44 @@ def test_get_all_acoustic_fields_filter_by_tablename(dbsession):
     num_fields = dbsession.query(AcousticField).count()
     num_main_fields = len(get_all_acoustic_fields(dbsession, tablename="main"))
     assert num_fields > num_main_fields
+
+
+def test_create_acoustic_newsletters_mapping(dbsession, acoustic_newsletters_mapping):
+    new_mapping = create_acoustic_newsletters_mapping(dbsession, "test", "sub_test")
+    assert (new_mapping.source, new_mapping.destination) == ("test", "sub_test")
+    all_mappings_count = dbsession.query(AcousticNewsletterMapping).count()
+    assert all_mappings_count > len(acoustic_newsletters_mapping)
+
+
+def test_create_acoustic_newsletters_mapping_duplicate_mapping(dbsession):
+    create_acoustic_newsletters_mapping(dbsession, "test", "sub_test")
+    with pytest.raises(sqlalchemy.exc.IntegrityError):
+        create_acoustic_newsletters_mapping(dbsession, "test", "sub_test")
+
+
+def test_create_acoustic_newsletters_mapping_source_to_many_dest(dbsession):
+    create_acoustic_newsletters_mapping(dbsession, "test", "sub_test")
+    create_acoustic_newsletters_mapping(dbsession, "test2", "sub_test")
+
+
+def test_delete_acoustic_newsletters_mapping(dbsession, acoustic_newsletters_mapping):
+    dbsession.add(AcousticNewsletterMapping(source="test", destination="sub_test"))
+    dbsession.commit()
+
+    deleted_mapping = delete_acoustic_newsletters_mapping(dbsession, "test")
+
+    assert (deleted_mapping.source, deleted_mapping.destination) == ("test", "sub_test")
+    all_mappings_count = dbsession.query(AcousticNewsletterMapping).count()
+    assert all_mappings_count == len(acoustic_newsletters_mapping)
```

Comment:
micro-nitpick: To follow the pattern you had for additions, you could have deleted one of `acoustic_newsletters_mapping` and asserted that count is `<`

Reply:
[655f2d7](https://github.com/mozilla-it/ctms-api/pull/599/commits/655f2d749f7cc261e10594bfab3a49ef68cf74bd)